### PR TITLE
MAINT: implement _wrap_derivative_exog for de-duplication

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: statsmodels/discrete/discrete_model.py [hunk 1]
```diff
@@ -406,12 +406,29 @@ def predict(self, params, exog=None, linear=False):
...
 
     def _derivative_exog(self, params, exog=None, dummy_idx=None,
-            count_idx=None):
+                         count_idx=None):
         """
         This should implement the derivative of the non-linear function
         """
         raise NotImplementedError
 
+    def _wrap_derivative_exog(self, margeff, params, exog, dummy_idx,
```
Comment (josef-pkt) [inline statsmodels/discrete/discrete_model.py]: "wrap" is not a good a good name because we use "wrap"ping mainly for pandas and output wrapping

**Comment by josef-pkt** [comment]
looks basically fine to me
For my taste there are already too many helper functions in margins that makes it difficult to figure out what is actually done for a specific case, e.g. the dummy effect.
postponed to 0.11 and to a hopefully more comprehensive review of margins.

File: statsmodels/discrete/discrete_model.py (hunk 1 above)
Comment (jbrockmendel) [reply statsmodels/discrete/discrete_model.py]: I'm open to suggestions

Comment (bashtage) [reply statsmodels/discrete/discrete_model.py]: _derivative_exog_helper

Comment (bashtage) [reply statsmodels/discrete/discrete_model.py]: _derivative_exog_formatter

Comment (bashtage) [reply statsmodels/discrete/discrete_model.py]: _derivative_exog_packager

**Pull request merged**